curity-sgnl-kong/sgnl_policy/patient.json

```diff
@@ -0,0 +1,27 @@
+{
+  "condition": {
+    "predicate": {
+      "lhs": {
+        "attributeId": "60da492f-bb33-4bed-b7ce-0f27aa6a9bee",
```

Comment:
suggest you placeholder these (attributeId/EntityId) so it's clear what they should be doing. The UUID isn't super helpful to help someone repro this.

Reply:
Done!